src/ostree/ot-builtin-refs.c

```diff
@@ -183,22 +207,35 @@ static gboolean do_ref (OstreeRepo *repo, const char *refspec_prefix, GCancellab
           else goto out;
         }
 
-      if (checksum_existing != NULL)
+      const gboolean creating_alias = opt_alias && opt_create;
+      if (!creating_alias || !g_hash_table_contains (ref_aliases, opt_create))
```

Comment:
We don't need `creating_alias` here, right? We're already in the `opt_create` branch, so the **if** condition can just be `(!opt_alias || !g_hash_table_contains (...))`, right?

Also, can we add a comment here re. `g_hash_table_contains`? It's a bit tricky to figure out the logic. E.g. even something simple like: `allow aliases to silently replace other aliases, but not regular refs`